docs/source/overview.rst

```diff
@@ -85,39 +80,38 @@ metadata integrity.
-functionality, a distributed network of collaborators can access, provide metadata
-for, and extract metadata from a collection of DataLad datasets.
-
-After access, the first step is to add any and all additional metadata to the
-DataLad dataset. This could either be standardized metadata such as a `datacite.xml`
-file, or a basic README or unstandardized JSON file. This is followed by metadata
-extraction from the dataset and its files using DataLad MetaLad together with
-built-in or extensible metadata extractors.
-
-When exported, all metadata objects can be passed to DataLad Catalog. This package
-translates the DataLad-generated metadata into structured data, which in turn is
-required by the `VueJS`_-based data browser interface. DataLad Catalog also
-generates the assets for the interface (artwork, CSS, JavaScript and HTML).
-
-The resulting user-friendly catalog can be hosted as a standalone web application
-without requiring any prior building steps or complicated infrastructure.
+DataLad Catalog can receive commands to ``create`` a new catalog, ``add`` and
+``remove`` metadata entries to/from an existing catalog, ``serve`` an existing
+catalog locally, and more. Metadata can be provided to DataLad Catalog from any
+number of arbitrary metadata sources, as an aggregated set or as individual
+items/objects. DataLad Catalog has a dedicated schema (using the `JSON Schema`_
+vocabulary) against which incoming metadata items are validated. This schema
+allows for standard metadata fields as one would expect for datasets of any kind
+(such as ``name``, ``doi``, ``url``, ``description``, ``license``, ``authors``,
+and more), as well as fields that support identification, versioning, dataset
+context and linkage, and file tree specification.
+
+The process of generating a catalog, after metadata entry validation, involves:
+
+1. aggregation of the provided metadata into the catalog filetree
+2. generating the assets required to render the user interface in a browser
+
+The output is a set of structured metadata files, as well as a `Vue.js`_-based
+browser interface that understands how to render this metadata in the browser.
+What is left for the user is to host this content on their platform of choice
+and to serve it for the world to see.
+
+For an example of the result, visit our `demo catalog`_.
+
+.. image:: /_static/datacat4_the_catalog.svg
+
+
 
 .. note:: A detailed description of these steps can be found in the :doc:`pipeline_description`
 
 .. _DataLad: https://github.com/datalad/datalad
 .. _DataLad Metalad: https://github.com/datalad/datalad-metalad
-.. _git: https://git-scm.com/
-.. _git-annex: https://git-annex.branchable.com/
-.. _VueJS: https://vuejs.org/
+.. _JSON Schema: https://json-schema.org/
+.. _Vue.js: https://vuejs.org/
+.. _demo catalog: https://datalad.github.io/datalad-catalog/#/dataset/deabeb9b-7a37-4062-a1e0-8fcef7909609/6d7fb68264f9b9951ae141fc830712a8744e3293
```

Comment:
```suggestion
.. _demo catalog: https://datalad.github.io/datalad-catalog
```

We can remove the reference to a specific dataset id and version since the catalog page will automatically redirect to whichever dataset id and version is set as the super-dataset of the catalog. This will prevent an error if e.g. this specific id and version are not contained in the demo catalog in future.

Reply:
Ah, nice! Thanks for pointing that out. I pushed the change.